### PR TITLE
Convenience methods for AssetTransfer

### PR DESCRIPTION
**Pull request by claudemiller**
Added methods for constructing an asset transfer with a single output given by assetId, amount, and receiving account. Added related tests. Related to #29.

### CODE REVIEW

File: wallet/src/test/java/io/neow3j/wallet/AssetTransferTest.java [hunk 1]
```diff
@@ -122,6 +122,196 @@ public void test_transfer_with_fee() throws IOException, ErrorResponseException
...
+                    .amount(BigDecimal.ONE)
+                    .toAddress("AJQ6FoaSXDFzA6wLnyZ1nFN7SGSN2oNTc3")
+                    .build();
+        } catch(IllegalStateException e) {
+            return;
+        }
+        fail();
+    }
+
+    @Test
```
Comment (gsmachado) [inline wallet/src/test/java/io/neow3j/wallet/AssetTransferTest.java]: Can you please adapt all the tests that you expect an exception with:
@Test(expect = SpecificException.class)

File: wallet/src/test/java/io/neow3j/wallet/AssetTransferTest.java [hunk 2]
```diff
@@ -122,6 +122,196 @@ public void test_transfer_with_fee() throws IOException, ErrorResponseException
...
+                    .asset(NEOAsset.HASH_ID)
+                    .output(output)
+                    .build();
+        } catch(IllegalStateException e) {
+            return;
+        }
+        fail();
+    }
+
+    @Test
```
Comment (gsmachado) [inline wallet/src/test/java/io/neow3j/wallet/AssetTransferTest.java]: same here!
The catch clause is not required.

File: wallet/src/test/java/io/neow3j/wallet/AssetTransferTest.java (hunk 1 above)
Comment (claudemiller) [reply wallet/src/test/java/io/neow3j/wallet/AssetTransferTest.java]: Done

File: wallet/src/test/java/io/neow3j/wallet/AssetTransferTest.java (hunk 2 above)
Comment (claudemiller) [reply wallet/src/test/java/io/neow3j/wallet/AssetTransferTest.java]: Done

**Pull request merged**